### PR TITLE
Add ETH goerli

### PR DESCRIPTION
**Pull request by hzheng-ledger**
Support ETH Goerli testnet in LL
https://ledgerhq.atlassian.net/browse/LIVE-1766

Screenshot 2022-03-29 at 15 22 07
Screenshot 2022-03-29 at 15 22 19

### CODE REVIEW

File: src/api/explorerConfig/index.ts
Comment (haammar-ledger) [inline src/api/explorerConfig/index.ts:195]: This is fine for testing a build, but we'll need a prod explorer before merging I guess.

**Comment by haammar-ledger** [comment]
@hzheng-ledger Also we could add ethereum_goerli to the tests, right?
I think it means adding it [here](https://github.com/LedgerHQ/ledger-live-common/blob/feat/LL-1766/src/__tests__/test-helpers/setup.ts#L53) and [here](https://github.com/LedgerHQ/ledger-live-common/blob/feat/LL-1766/src/mock/account.ts#L400) and see if tests pass or if any test needs to be updated.

**Comment by hzheng-ledger** [comment]
> @hzheng-ledger Also we could add ethereum_goerli to the tests, right? I think it means adding it [here](https://github.com/LedgerHQ/ledger-live-common/blob/feat/LL-1766/src/__tests__/test-helpers/setup.ts#L53) and [here](https://github.com/LedgerHQ/ledger-live-common/blob/feat/LL-1766/src/mock/account.ts#L400) and see if tests pass or if any test needs to be updated.

I added it into them now. waiting for CI now :)

**Review by ghost — APPROVED** [review]

**Pull request merged**
